src/Uno.UWP/Storage/Helpers/StorageFile.skia.cs

```diff
@@ -0,0 +1,29 @@
+using System.Threading;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Windows.Storage.Helpers
+{
+	partial class StorageFileHelper
+	{
+		private static Task<bool> FileExistsInPackage(string filename)
+		{
+			var executingPath = Assembly.GetExecutingAssembly().Location;
+			if (!string.IsNullOrWhiteSpace(executingPath))
+			{
+				var path = Path.GetDirectoryName(executingPath);
+				if (path is not null &&
+					!string.IsNullOrWhiteSpace(path))
```

Comment:
```suggestion
				if (!string.IsNullOrEmpty(path))
```

Reply:
IsNullOrWhiteSpace is broader test than IsNullOrEmpty, so why such change?

Reply:
The broader test is unnecessary and doesn't serve any purpose.